packages/frontend/core/src/blocksuite/ai/settings/indexer-embedding/entities/embedding.ts

```diff
@@ -0,0 +1,242 @@
+import { logger } from '@affine/core/modules/share-doc/entities/share-docs-list';
```

Comment:
Better to move services and entities to fold packages/frontend/core/src/modules?

Reply:
moved